ipatests/test_integration/test_trust.py

```diff
@@ -463,6 +357,50 @@ def test_user_uid_resolution_in_external_trust_with_shared_secret(self):
         assert re.search(
             testuser_regex, result.stdout_text), result.stdout_text
 
+    def test_override_homedir(self):
+        """
```

Comment:
As per https://www.freeipa.org/page/Python_Coding_Style docstrings should comply PEP257 (https://www.python.org/dev/peps/pep-0257/#id17):
 * docstring should start on same line as quotes
 * first line should shortly describe the method
 * 'The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".'
 * There must be an empty line between first line and remaining lines.